addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -3929,8 +3929,167 @@ registry.SnippetSave = SnippetOptionWidget.extend({
     },
 });
 
+/**
+ * Allows to copy the anchor to clipboard and optionally rename the anchor.
+ */
+const CopyAnchorToClipboardAndNotifyWidget = SnippetOptionWidget.extend({
+
+    /**
+     * @override
+     * @param {jQuery} $trigger the element that triggers the copy to clipboard and the notification
+     */
+    start: function ($trigger) {
+        if ($trigger) {
```

Comment:
Hum... how is this possible? `start` is a base method of the Widget class, it does not receive a parameter so it seems that the rest of this method does not ever do anything? Waiting your answer on that before continuing the review, maybe it is worth you check is still works and rebase it too :+1:

Reply:
@qsm-odoo It's possible because he made it possible ;) Look at the code in the start method of the snippet popup options. the parameter is added in the return => `return this._super.apply(this, dataAttributeDisplayOnClick || $(), arguments);`

Reply:
Yes he told me :wink: Needs to be changed to avoid that (with a simple write in a property in `this.`). As LTU works elsewhere, I told him you can take the branch again ;)

Reply:
Hello @bvr-odoo !
@qsb-odoo is right :o) I should have instead given a value to this.$trigger the call start of super.

Reply:
@IT-Ideas You left the team a week ago and you already forgot my trigram :cry: :sob: 
cc @JKE-be

Reply:
You know me 😉  I went too fast 🚗

Reply:
![image](https://user-images.githubusercontent.com/4848797/95373520-67f77f80-08dd-11eb-8bef-cd88dfe81798.png)

Reply:
😄

Reply:
[![Tu as deja oublié la website Team ?????????](http://img.youtube.com/vi/pZw2nRg1a-s/0.jpg)](https://www.youtube.com/watch?v=pZw2nRg1a-s&feature=youtu.be&t=64 "Tu m'as oublié")

Reply:
[https://www.youtube.com/watch?v=W_XEr3mH0Rc](url)

Reply:
@qsm-odoo I made the change